docs/Makefile

```diff
@@ -49,6 +49,7 @@ node_modules:
 	npm config set registry http://registry.npmjs.org/
```

Comment:
Do we actually need this line?  My other node apps have a package.json file that defines the dependencies and maybe that takes care of this config setting for me.